src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,16 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => {
+  if (comments.length === 0) {
+    return <div data-cy="NoCommentsMessage">No comments yet.</div>;
+  }
```

Comment:
CommentList should just render a list of comments. Move this condition at upper scope to (PostInfo)